src/super_gradients/training/transforms/processing.py

```diff
@@ -204,19 +248,10 @@ def postprocess_predictions(self, predictions: DetectionPrediction, metadata: Re
         return predictions
 
 
-def get_pretrained_processing_params(model_name: str, pretrained_weights: str) -> Tuple[Optional[List[str]], Optional[Processing]]:
-    """Get the processing parameters for a pretrained model."""
-    if "yolox" in model_name and pretrained_weights == "coco":
-        return default_yolox_coco_processing_params()
-    elif "ppyoloe" in model_name and pretrained_weights == "coco":
-        return default_ppyoloe_coco_processing_params()
-    else:
-        return None, None
-
-
-def default_yolox_coco_processing_params() -> Tuple[List[str], Processing]:
-    """Processing parameters commonly used for training YoloX on COCO dataset."""
-    from super_gradients.training.datasets.datasets_conf import COCO_DETECTION_CLASSES_LIST
+def default_yolox_coco_processing_params() -> dict:
+    """Processing parameters commonly used for training YoloX on COCO dataset.
+    TODO: remove once we load it from the checkpoint
+    """
 
     image_processor = ComposeProcessing(
         [
```

Comment:
This part and the code below (within the processing.py module) were actually merge to deciyolo already.
The thing is that I believe that we should merge first to master because otherwise the deciyolo branch would have lots of code to review at once, so I chose to merge this branch to master first, and I took with me some required code that was already in deciyolo branch.

Reply:
Make sense to me